AnmolSirola/echidna
# Update README.md

## Description

This pull request addresses issue #1063 and enhances the documentation for the Echidna tool by adding instructions on how to leverage the lcov support with a VSCode plugin. The updated documentation provides clear and beginner-friendly steps to install the lcov plugin, generate coverage reports, open the reports in VSCode, and utilize the lcov plugin to visualize the coverage of smart contracts.

## Changes Made

- Added a new section to the documentation titled "Using lcov Support with a VSCode Plugin in Echidna."
- Included step-by-step instructions with code snippets and explanations for each step.
- Formatted the instructions in a clear and concise manner using Markdown.
- Provided an overview of the lcov plugin's features for visualizing coverage, such as the coverage map, call graph, and function coverage summary.

## How to Test

1. Follow the instructions provided in the updated documentation to install the lcov plugin for VSCode.
2. Use the provided steps to run Echidna and generate a coverage report.
3. Open the coverage report in VSCode and verify that the lcov plugin automatically visualizes the coverage of the smart contracts.
4. Explore the lcov plugin's features, such as the coverage map, call graph, and function coverage summary, to validate their functionality.

## Related Issues

Closes #1063


Please let me know if there are any further modifications or additions required. I'm happy to assist!
